Description: Don't install compatibility symlinks
 The project has been renamed upstream more than two years ago,
 and the transitional binary package that used to provide
 compatibility with the old name is no longer being built.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2014-08-18
Index: spectrwm.git/linux/Makefile
===================================================================
--- spectrwm.git.orig/linux/Makefile	2016-06-13 00:09:44.379604210 +0200
+++ spectrwm.git/linux/Makefile	2016-06-13 00:09:44.375604144 +0200
@@ -47,7 +47,6 @@
 	install -m 755 -d $(DESTDIR)$(MANDIR)/ru/man1
 	install -m 755 -d $(DESTDIR)$(XSESSIONSDIR)
 	install -m 755 spectrwm                    $(DESTDIR)$(BINDIR)
-	ln -sf spectrwm                            $(DESTDIR)$(BINDIR)/scrotwm
 	install -m 644 libswmhack.so.$(LIBVERSION) $(DESTDIR)$(LIBDIR)
 	ln -sf libswmhack.so.$(LIBVERSION)         $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBMAJORVERSION)
 	ln -sf libswmhack.so.$(LIBVERSION)         $(DESTDIR)$(LIBDIR)/libswmhack.so
@@ -60,7 +59,6 @@
 
 uninstall:
 	rm -f $(DESTDIR)$(BINDIR)/spectrwm
-	rm -f $(DESTDIR)$(BINDIR)/scrotwm
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBVERSION)
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBMAJORVERSION)
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so
